Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -50,19 +51,34 @@ final class PledgePaymentMethodsViewController: UIViewController {
     )
   }
 
+  override func viewDidLayoutSubviews() {
+    super.viewDidLayoutSubviews()
+
+    self.updateScrollViewHeightConstraint()
+  }
+
+  override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
```

Comment:
I *think* you might not need this because `viewDidLayoutSubviews` gets called when the trait collection changes. 🤔

Reply:
Actually we need this, otherwise the scrollView won't show. The same apply for the `viewDidLayoutSubviews`